src/Controller/Organization/PackageController.php

```diff
@@ -110,6 +113,52 @@ public function packageNew(Organization $organization, Request $request, ?string
         ]);
     }
 
+    /**
+     * @Route("/organization/{organization}/package/{package}", name="organization_package_update", methods={"POST"}, requirements={"organization"="%organization_pattern%","package"="%uuid_pattern%"})
+     */
+    public function updatePackage(Organization $organization, Package $package): Response
+    {
+        $this->dispatchMessage(new SynchronizePackage($package->id()));
+
+        $this->addFlash('success', 'Package will be updated in the background');
+
+        return $this->redirectToRoute('organization_packages', ['organization' => $organization->alias()]);
+    }
+
+    /**
+     * @Route("/organization/{organization}/package/{package}/edit", name="organization_package_edit", methods={"GET","POST"}, requirements={"organization"="%organization_pattern%","package"="%uuid_pattern%"})
+     */
+    public function editPackage(Organization $organization, Package $package, Request $request): Response
+    {
+        $form = $this->createForm(EditPackageType::class, [
+            'url' => $package->url(),
+            'keepLastReleases' => $package->keepLastReleases(),
+        ]);
+
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            $data = $form->getData();
+
+            $this->dispatchMessage(new Update(
+                $package->id(),
+                $data['url'],
+                $data['keepLastReleases'],
+            ));
+
+            $this->dispatchMessage(new SynchronizePackage($package->id()));
+
+            $this->addFlash('success', 'Package will be updated in the background');
```

Comment:
`Package will be updated in the background`, maybe we should slightly modify this message, what about: `Package will be synchronized in the background`